components/ViewArticleListPage/ArticleDataUtils.ts

```diff
@@ -0,0 +1,41 @@
+export interface ArticleRowData {
```

Comment:
this file u can consider moving it to the models folder. there is one for campaigns already then u can create one for articles

Reply:
also can refer to the articleSchema in the backend repo for the type of article and what can be filtered. https://github.com/dsc-project-mr/mr-backend/blob/main/src/models/content.model.ts

Reply:
from what i see, i think can do filtering on state, type, tags and date